rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/core/RestControllerHelper.java

```diff
@@ -330,6 +330,16 @@ public void addEmbeddedProject(HalResource halResource, Project project) {
         halResource.addEmbeddedResource("sw360:projects", halProject);
     }
 
+    public Project updateProject(Project projectToUpdate, Project requestBodyProject) {
+        for(Project._Fields field:Project._Fields.values()) {
+            Object fieldValue = requestBodyProject.getFieldValue(field);
+            if(fieldValue != null) {
```

Comment:
I know that you implemented this logic the same way like it is for Release and Component. But do you have any idea why we wouldn't want to let the user reset fields (set them to null)?
Would it maybe be helpful to check with `isSet(Field)` - even though I cannot tell how this would work out with spring deserialization.

Reply:
Well, the `isSet(Field)` seems to do nothing else then a null check... too bad. So I have no other spontaneous idea how to actually enable users to reset a value.

Reply:
As you said, the logic is implemented the same was as we have for Release and Component. I agree this needs to be improved for all update operation . Currently if user wants to reset the field then he can pass "" string . if the field is not passed in the request json then that field will be ignored.